app/org/sagebionetworks/bridge/dao/SurveyDao.java

```diff
@@ -48,7 +48,16 @@
      * @param keys
      */
     public void deleteSurvey(GuidCreatedOnVersionHolder keys);
-    
+
+    /**
+     * Admin API to remove the survey from the backing store. This exists to clean up surveys from tests. This will
+     * remove the survey regardless of publish status, whether it has responses. This will delete all survey elements
+     * as well.
+     *
+     * @param keys survey keys (guid, created-on timestamp)
+     */
+    public void deleteSurveyPermanently(GuidCreatedOnVersionHolder keys);
```

Comment:
Thanks, this was bothering me. The tables are filling up with tons and tons of test data